tests/keras/backend/mxnet_sparse_test.py

```diff
@@ -104,6 +105,43 @@ def test_sparse_dot(self):
         assert k_s.shape == k_d.shape
         assert_allclose(k_s, k_d, atol=1e-05)
 
+    def _forward_pass(self, x):
+        bind_values = K.dfs_get_bind_values(x)
+        executor = x.symbol.simple_bind(mx.cpu(), grad_req='null')
+        for v in executor.arg_dict:
+            bind_values[v].copyto(executor.arg_dict[v])
+        outputs = executor.forward(is_train=K.learning_phase())
+        return outputs
+
+    def test_sparse_embedding(self):
```

Comment:
Could you add a test similar to tests/keras/layers/embedding_test.py? use  layer_test to test Embedding layer class